frontend-mobile/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -5,6 +5,8 @@
 import FlutterMacOS
 import Foundation
 
+import shared_preferences_macos
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
+  SharedPreferencesPlugin.register(with: registry.registrar(forPlugin: "SharedPreferencesPlugin"))
```

Comment:
Reading the docs on my cell phone I didn't see this. Why do you think it's necessary?

Reply:
Oh wait, this was autogenerated, because you ran it on MacOs, right?

Reply:
I think so yeah